packages/react/src/components/Tag/Tag-test.js

```diff
@@ -67,6 +68,30 @@ describe('Tag', () => {
     ).toBeInTheDocument();
   });
 
+  describe('Selectable Tag', () => {
+    it('should render a selectable tag', () => {
+      const { container } = render(<SelectableTag text="Tag content" />);
+
+      const selectableTag = container.querySelector(
+        `.${prefix}--tag--selectable`
+      );
```

Comment:
I commented this in another PR too, but would it make sense to use `container.firstChild` instead?